src/schemas/json/putout.json

```diff
@@ -0,0 +1,247 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "definitions": {
+    "definitions": {
+      "rule": {
```

Comment:
@coderaiser There is one "definitions" level too many.

Reply:
Thank you! Just removed.